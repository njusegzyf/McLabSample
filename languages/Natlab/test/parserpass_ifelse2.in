if  n < 0   
else
end
