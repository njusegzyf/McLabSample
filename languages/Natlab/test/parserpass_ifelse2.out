1 1
3 5
if (n < 0)
else 
end
